Makefile

```diff
@@ -266,6 +276,14 @@ set-version:
 	@sed -i".out" -e 's@Version[ ]*=.*@Version = "$(VERSION)"@g' ./version/version.go;
 	rm -rf ./version/version.go.out
 
+.PHONY: generate-scalers-schema
+generate-scalers-schema: ## Generate scalers shcema
```

Comment:
We should call this in `generate` probably, so the schema is regenerated with each change.